app/bundles/PageBundle/Controller/PublicController.php

```diff
@@ -365,9 +365,15 @@ public function previewAction($id)
      */
     public function trackingImageAction()
     {
+         /** @var LeadModel $leadModel */
+        $leadModel = $this->getModel('lead');
+
+        $lead = $leadModel->getCurrentLead();
+        
         /** @var \Mautic\PageBundle\Model\PageModel $model */
         $model = $this->getModel('page');
-        $model->hitPage(null, $this->request);
+        //$model->hitPage(null, $this->request);
+        $model->hitPage(null, $this->request, '200', $lead);
```

Comment:
Passing `$lead` completely bypasses the logic in [PageModel](https://github.com/mautic/mautic/blob/staging/app/bundles/PageBundle/Model/PageModel.php#L462-L479). This alters current application behaviour. E.g. `ct` query parameter is not taken into account, etc.